vulnerabilities/static/css/custom.css

```diff
@@ -6,4 +6,8 @@
 
 .Site-content {
 flex: 1;
+}
+
+table {
```

Comment:
I'm guessing this won't  be needed if we use bulma ?

Reply:
This was necessary. I removed the `is-bordered` class from the table so that cells are not divided by a line. That was causing difficulty in understanding the structure of severity